### PR TITLE
chore: modific generate add skip exist files and fix paths

### PR DESCRIPTION
**Pull request by italobarrosme**
## Description
Added parameter to skip file creation if the file already exists, 
fixed paths to components and 
modified generate script plop
## Screenshot
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "prepare": "husky install",
-    "generate": "yarn workspace @mesa/generators generate"
+    "generate:component:vue": "yarn workspace @mesa/generators generate"
```
Comment (christiantld) [inline package.json]: I believe we could keep the script as `generate`  and update the plop setGenerator name to something related to what will be generated, like `vue-component`or `component:vue`

File: actions/generators/src/plopfile.js [hunk 2]
```diff
@@ -15,28 +15,33 @@ export default function (
...
         type: 'add',
-        path: '../src/components/{{pascalCase name}}/{{pascalCase name}}.stories.mdx',
+        path: '../../../packages/vue/src/components/{{pascalCase name}}/{{pascalCase name}}.stories.mdx',
         templateFile: 'templates/[stories].mdx.hbs',
+        skipIfExists: true,
       },
       {
         type: 'add',
-        path: './../styles/components/{{pascalCase name}}.module.css',
+        path: '../../../packages/vue/src/components/{{pascalCase name}}/{{pascalCase name}}.module.css',
```
Comment (christiantld) [inline actions/generators/src/plopfile.js]: As we discuss the styles and types will be part of its own package so maybe the relative path need to be changed, ex: `path: '../../../packages/styles/...`

**Review by christiantld — CHANGES_REQUESTED** [review]
The point related to the relative path is something that we will have to update

File: package.json (hunk 1 above)
Comment (italobarrosme) [reply package.json]: ✔️

File: actions/generators/src/plopfile.js (hunk 2 above)
Comment (italobarrosme) [reply actions/generators/src/plopfile.js]: ✔️

**Review by christiantld — APPROVED** [review]

**Pull request merged**